phpstan.neon

```diff
@@ -9,5 +9,6 @@ parameters:
         # Some parent constructors are explicitly to be ignored
         - '#does not call parent constructor#'
         - '#Access to an undefined property PhpParser\\Node\\Param::\$isOptional#'
+        - '#Access to an undefined property PhpParser\\Node\\Expr::\$value#'
         # Impossible to define type hint for anonymous class
-        - '#Call to an undefined method PhpParser\\NodeVisitorAbstract::(clearNodes|getClassNodes|getFunctionNodes)\(\)#'
+        - '#Call to an undefined method PhpParser\\NodeVisitorAbstract::(clearNodes|getClassNodes|getFunctionNodes|getConstantNodes|getNode|setConstantName)\(\)#'
```

Comment:
Can we limit these exclusions to just the stub map generator

Reply:
Done.